src/components/UserInfo.js

```diff
@@ -11,8 +12,12 @@ export default class UserInfo {
     };
   }
 
-  setUserInfo(data) {
-    this._profileTitle.textContent = data.title;
-    this._profileDescription.textContent = data.description;
+  setUserInfo(userInfoData) {
+    this._profileTitle.innerHTML = userInfoData.name;
```

Comment:
`innerHTML` should not be used because it has some `vulnerability`. Please, use `textContent` for any text

**Please, correct it everywhere**

Reply:
You haven't corrected one more remark https://github.com/Parnika006/se_project_aroundtheus/pull/1#discussion_r1693274530